LayoutTests/svg/text/textLength-tspan-1-expected.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+<body>
+<h1>All lines should have the same length</h1>
+<svg style="display: block; width: 100%; height: 600px;" viewBox="0 0 100 100">
```

Comment:
My be more natural for SVG to have separate width height PA here.

Reply:
Really? These days I thought people move towards this style when using inline SVG...